pkg/test/resourcefixture/contexts/sql_context.go

```diff
@@ -59,13 +59,34 @@ func init() {
 		ResourceKind:  "SQLInstance",
 	}
 
+	resourceContextMap["sqlinstanceauthorizednetworks"] = ResourceContext{
```

Comment:
Do you understand what this does?  (I don't!)

I wonder if we should have this functionality in the unified_test.